src/modal/section/pending-message/PeraWalletConnectModalPendingMessage.ts

```diff
@@ -147,6 +150,22 @@ export class PeraWalletConnectModalPendingMessageSection extends HTMLElement {
     removeModalWrapperFromDOM(PERA_WALLET_CONNECT_MODAL_ID);
   }
 
+  addAudioForConnection() {
```

Comment:
why you didn't directly use `audio` element in HTML?

Reply:
I can't hide on different devices. Because audio tag doesn't have attribute style.